code/modules/projectiles/guns/launcher/crossbow.dm

```diff
@@ -57,15 +57,15 @@
 	slot_flags = SLOT_BACK
 	restrict_safety = TRUE
 	twohanded = TRUE
-	rarity_value = 48
+	rarity_value = GUN_BASE_RARITY_VALUE+30//no price tag,then high rarirty
 
 	var/obj/item/bolt
-	var/tension = 0                         // Current draw on the bow.
-	var/max_tension = 5                     // Highest possible tension.
-	var/release_speed = 5                   // Speed per unit of tension.
-	var/obj/item/weapon/cell/large/cell    // Used for firing superheated rods.
-	var/current_user                        // Used to check if the crossbow has changed hands since being drawn.
-	var/draw_time = 20							// How long it takes to increase the draw on the bow by one "tension"
+	var/tension = 0						// Current draw on the bow.
```

Comment:
What is going on with these tabbing changes from line 63 to 67?

Reply:
It seemed to me that the comments would be easier to read if they were in the same column.

![image](https://user-images.githubusercontent.com/49106866/107466077-d5553e80-6b63-11eb-9e98-dc15b29f2c06.png)